src/blocks/_pro/post-category-badge/index.php

```diff
@@ -92,6 +92,10 @@ function vk_blocks_post_category_badge_build_variations() {
 
 	// Create and register the eligible taxonomies variations.
 	foreach ( $taxonomies as $taxonomy ) {
+		if ( ! $taxonomy->hierarchical && 'post_tag' === $taxonomy->name ) {
```

Comment:
@akito-38 今回の変更って post_tag を除外するだけなので↓だけでよくないですか（・ｗ・？ 

```suggestion
		if ( 'post_tag' === $taxonomy->name ) {
```